pkg/notes/notes.go

```diff
@@ -416,33 +438,45 @@ func ListCommitsWithNotes(
 	}
 
 	for _, commit := range commits {
+
+		level.Debug(logger).Log("msg", "################################################" )
```

Comment:
Please avoid those separating log messages :)

Reply:
It was tricky to see where the parsing/processing for a specific commit was happening with all the messages. This just made it visually easier. These are all debug log level messages so aimed to make debugging easier. In standard running mode I should check that these are not output.

Thoughts?

Reply:
There's some prior art (albeit in the Bash) that breaks up some logs with separators for debugging. I think it should be fine so long as it's not output on a normal run.